benchmark/flink-kafka/docker-compose.yml

```diff
@@ -2,7 +2,9 @@ version: "2.2"
 name: nexmark
 services:
   jobmanager:
-    image: localhost/nexmark-kafka:latest
+    build:
+      context: .
+    image: nexmark-kafka:latest
```

Comment:
I think that `build` and `image` are mostly exclusive; docker compose picks one of them. Do we want both?

Reply:
Removing "image" seems to fail to copy some files like run-nexmark.sh to the container.
Removing "build" means that the container's image will not be automatically built. However, when "build" is included, the image is not rebuilt when files change. I think the best way is to remove "build" and keep the "docker build" command which you commented on below since it rebuilds the image whenever run.

Reply:
Ugh. Thanks for figuring it out.